polaris.shopify.com/scripts/gen-site-map.mjs

```diff
@@ -1,21 +1,41 @@
-import {execa} from 'execa';
-import path from 'path';
+import {prettyExeca, startLocalServer} from './util.mjs';
 
 const genSiteMap = async () => {
   const outputFile = 'public/sitemap.xml';
 
-  const nextBin = path.join(process.cwd(), 'node_modules/.bin/next');
-  const server = execa(nextBin, ['dev']);
+  const server = startLocalServer('dev');
+  server.catch((err) => {
+    if (!server.killed) {
```

Comment:
This check results in less noisy output when the process fails for whatever reason.